packages/cli/src/commands/list.js

```diff
@@ -1,11 +1,18 @@
 /* global process */
 import os from 'os';
 import { E } from '@endo/far';
-import { withEndoParty } from '../context.js';
+import { withEndoHost } from '../context.js';
 
-export const list = async ({ partyNames }) =>
-  withEndoParty(partyNames, { os, process }, async ({ party }) => {
-    const petNames = await E(party).list();
+export const list = async ({ directoryName, special, all }) =>
+  withEndoHost({ os, process }, async ({ host: party }) => {
+    if (directoryName !== undefined) {
+      party = E(party).lookup(directoryName);
+    }
+    const petNames = await (() => {
+      if (all) return E(party).listAll();
+      if (special) return E(party).listSpecial();
```

Comment:
Do the `--all` and `--special` flags work? My preference is to eliminate them and just list special names with the usual `list()`.